orga/translations/en.json

```diff
@@ -309,6 +311,28 @@
         "invalid-division": "The class {selectedDivision} does not exist."
       }
     },
+    "login": {
+      "title": "Log in"
+    },
+    "login-form": {
+      "active-or-retrieve": "",
+      "email": "Email address",
+      "forgot-password": "Forgot your password?",
+      "forgotten-password-url": "https://app.pix.org/mot-de-passe-oublie?lang=en",
+      "invitation-already-accepted": "This invitation has already been accepted. Please log in or contact the administrator of your Pix Orga workspace.",
+      "is-only-accessible": "Pix Orga is only accessible to invited members. Please contact your Pix Orga administrator in order to be invited.",
+      "login": "Log in",
+      "only-for-admin": "",
```

Comment:
**Remarque** : Tu avais fait le choix de ne pas mettre la traduction pour `"active-or-retrieve"` et  `"only-for-admin"` pour qu'ils n'apparaissent pas en anglais. Mais avec la nouvelle correction (displayRecoverLink) qui masque la div si on est en `en`, je propose qu'on remette les traductions ? Sauf s'il n'y a aucune chance que cette partie redevienne visible si on est en anglais ?

Reply:
Done